### PR TITLE
React, babel, and webpack dependencies all installed and App rendering to b…

### PR DESCRIPTION
**Pull request by alishaedington**
…rowser

### CODE REVIEW

File: client/dist/bundle.js [hunk 1]
```diff
@@ -0,0 +1,256 @@
+/*
```
Comment (murkymode) [inline client/dist/bundle.js:1]: add to .gitignore

Comment (murkymode) [inline client/dist/bundle.js]: add to .gitignore

**Review by murkymode — CHANGES_REQUESTED** [review]

File: client/dist/bundle.js (hunk 1 above)
Comment (alishaedington) [reply client/dist/bundle.js]: i tried...

**Review by murkymode — APPROVED** [review]
Win!

**Pull request merged**